src/routers/users.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
Since you are using modern JavaScript features, you can remove 'use strict' as it's not needed in ES6 modules.

Reply:
But don't change here